hash/sega_beena_cart.xml

```diff
@@ -60,6 +206,7 @@ license:CC0
 		<description>Soreike! Anpanman Hajimete Kaketa yo! Oboeta yo! Hiragana Katakana - Gojuuon Board Kinou-tsuki</description>
 		<year>2005</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="それいけ！アンパンマン どきどき！レスキュードライブ ～カーナビつき～"/>
```

Comment:
That says “Soreike! Anpanman dokidoki! Rescue drive ~Car Navi tsuki~” – doesn’t match the title.

Reply:
Fixed, thanks!